mods/ts/rules/shared-vehicles.yaml

```diff
@@ -134,7 +134,7 @@ LPST:
 		RequiresCondition: !empdisable && undeployed
 	RevealsShroud:
 		RequiresCondition: !inside-tunnel
-		Range: 7c0
+		Range: 10c0
```

Comment:
According to the rules.ini i'm reading, the deployed version actually had less sight (8).  Is this correct?  If so, we'll want a `RevealsShroud@DEPLOYED`.